presto-product-tests/conf/presto/etc/catalog/mysql.properties

```diff
@@ -1,4 +1,4 @@
 connector.name=mysql
-connection-url=jdbc:mysql://mysql:3306/test
+connection-url=jdbc:mysql://mysql:13306/test
```

Comment:
There are two possibilities how to fix this problem:
1. remove that mapping and follow the same pattern as we do for Hive (1-to-1 mapping).
2. prepare different set of configurations for running from IntelliJ and on automation (possible issues of getting those two sets of configuration out of sync).

Any thoughts?
CC: @arhimondr, @ArturGajowy, @fiedukow